jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/details/ProcessInstanceDetailsViewImpl.java

```diff
@@ -43,6 +43,10 @@
     @Inject
     @DataField
     public HTML processVersionText;
+    
+    @Inject
+    @DataField
+    public HTML slaComplianceText;
```

Comment:
These text fields need fixing as they don't match the html element from the template (HTML vs Paragraph). If you could try to use _HTMLParagraphElement_ that would get us on the right path.